components/PralineList/PralineList.js

```diff
@@ -23,7 +23,7 @@ export default function PralineList({ onSelectPraline }) {
             />
           )}
           <button type="button" onClick={() => onSelectPraline(praline)}>
-            bearbeiten
+            {buttonName}
```

Comment:
This would change to children then.
```suggestion
            {children}
```